src/components/authorize/tests/authorize-controller.test.ts

```diff
@@ -108,7 +111,15 @@ describe("authorize controller", () => {
         fakeJwtService
       )(req as Request, res as Response);
 
-      expect(res.cookie).to.have.been.called;
+      expect(res.cookie).to.have.been.calledWith(
+        COOKIES_PREFERENCES_SET,
+        consentCookieValue.value,
+        sinon.match({
```

Comment:
```suggestion
        sinon.match({
        expires: sinon.match((date: Date) => {
            const expectedExpires = new Date(Date.now());
            expectedExpires.setFullYear(expectedExpires.getFullYear() + 1)
            return Math.abs(date.getTime() - expectedExpires.getTime()) < 1000;
          }),
```

Reply:
Check that the expiry date is approximately one year in the future from now allowing a difference of 1000 milliseconds to account for the time the test takes to run.